src/DateRangePicker.elm

```diff
@@ -532,10 +547,10 @@ panel toMsg (State internal) =
             [ span []
                 [ case internal.step of
                     Step.Initial ->
-                        text "Hint: pick a start date"
+                        text baseCalendar.translations.pickStart
 
                     Step.Begin _ ->
-                        text "Hint: pick an end date"
+                        text baseCalendar.translations.pickStart
```

Comment:
Hi :) I pulled your change locally, because I run into the same issue as you do, regarding the translations. 
There is a teensy bug on this line the `pickStart` here should be `pickEnd`. Hope this helps!